tools/spark-reader-v1/src/main/java/com/vesoft/nebula/bean/ConnectInfo.java

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.bean;
+
+import com.google.common.base.Preconditions;
+import com.google.common.net.HostAndPort;
+import com.vesoft.nebula.common.Checkable;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Data;
+import org.apache.commons.lang.StringUtils;
+
+@Data
```

Comment:
Using lombok requires developers to install the lombok plugin. So it is not recommended to use lombok.